python/tirex_tracker/__init__.py

```diff
@@ -464,21 +477,8 @@ def _get_python_info(
             results=results,
             measure=Measure.PYTHON_CODE_ARCHIVE_PATH,
             measures=measures,
-            value=str(archive_paths.zip_file_path.resolve()),
+            value=str(archive_paths.zip_file_path),
         )
-        _add_python_result_entry(
-            results=results,
-            measure=Measure.PYTHON_SCRIPT_FILE_PATH,
-            measures=measures,
-            value=str(archive_paths.script_file_path.resolve()),
-        )
-        if archive_paths.notebook_file_path is not None:
-            _add_python_result_entry(
-                results=results,
-                measure=Measure.PYTHON_NOTEBOOK_FILE_PATH,
-                measures=measures,
-                value=str(archive_paths.notebook_file_path.resolve()),
-            )
```

Comment:
Why are these lines removed?

Reply:
Because we currently can not detect what the PYTHON_NOTEBOOK_FILE_PATH is. We just had wrong values there.

Reply:
Alright, I would favor to completely remove the `PYTHON_SCRIPT_FILE_PATH` and `PYTHON_NOTEBOOK_FILE_PATH` measures then in favor of the `PYTHON_SCRIPT_FILE_PATH_IN_CODE_ARCHIVE` and `PYTHON_NOTEBOOK_FILE_PATH_IN_CODE_ARCHIVE` measures. These should be less confusing. No need to keep the confusing measures then.